logger.go

```
@@ -79,11 +82,54 @@ func setupTargetLogger(flags commandLineFlags, logTarget, commandOutput string)
+	err := w.LogCloser.Close()
+	if err != nil {
+		return err
+	}
+	// Open logfile for reading
+	logData, err := os.Open(w.logfilePath)
+	if err != nil {
+		return err
+	}
+	// Upload logfile to server
+	client := &http.Client{Timeout: 30 * time.Second}
+	resp, err := client.Post(w.logUploadTarget, "application/octet-stream", logData)
```

golangci-lint (1.64.8)

(*net/http.Client).Post must not be called

(noctx)